### DISCUSSION TITLE
Search on Microsoft SQL Server

### DISCUSSION THREAD

**Discussion by SirBamalot** [requarks/wiki#6666] [General]
We are using Wiki.js with MSSQL database engine. 
In Wiki.js search is configures as "Basic DB" which limits the search capabilities of the software to title and description. 

![image](https://github.com/requarks/wiki/assets/142885542/c390faa2-2af1-4e26-a3e8-7ba4bf0cd234)

To unleash the full power of Wiki.js search it looks like PostgreSQL is recommended. With PostgreSQL Wiki.js seems to offer full text search as well. 

Sadly PostgreSQL is not supported in our company. 
Is there a way to get full text search running with MS SQL Server as well?

**Comment by kjette** [comment]
PostgreSQL will only be the supported DB in wikijs v3, so you should try to convince your company to actually support it if you want to run wikijs in your organization. 

It is very simple to install and maintain PostgreSQL in my opinion. I recommend;

Ubuntu LTS (22.04 currently) with PostgreSQL 15. 

Look at the beginning of this guide for example:
https://www.linuxtechi.com/how-to-install-postgresql-on-ubuntu/
